src/metabase/db/custom_migrations.clj

```diff
@@ -91,3 +94,9 @@
   (t2.execute/query-one {:delete-from :permissions
                          :where [:or [:like :object (h2x/literal "/data/db/%")]
                                  [:like :object (h2x/literal "/query/db/%")]]}))
+
+(define-migration DeleteAbandonmentEmailTask
+  (task/start-scheduler!)
+  (task/delete-task! (jobs/key "metabase.task.abandonment-emails.job")
+                     (triggers/key "metabase.task.abandonment-emails.trigger"))
+  (task/stop-scheduler!))
```

Comment:
just to be "extra sure" should we put this into a try catch and run `stop-scheduler` in finally?

Reply:
hmm, I'm worried about this. Even though we know now that `start-scheduler!` is idempotent, but we should keep these custom migrations side-effect free.

In fact we would like to keep custom migrations to use as less as possible code from other ns because those functions could change over time.

See [this](https://metaboat.slack.com/archives/CKZEMT1MJ/p1675921528936249?thread_ts=1675892968.988579&cid=CKZEMT1MJ) from Cam

Reply:
Fair concerns, I will copy the relevant code from the task namespace into this namespace for extra stability, and create a separate scheduler to the one used by the app. That way we can start and stop the scheduler in the repl without side-effects.